jsonnet/config.jsonnet

```diff
@@ -111,13 +111,16 @@
             {
               name: 'Checkout main',
               uses: 'actions/checkout@v2',
+              with: {
+                'fetch-depth': 0,
+              },
             },
+            { uses: 'actions/setup-python@v2' },
+            { run: 'pip install -r requirements.txt' },
+            { run: "git config user.name 'github-actions[bot]' && git config user.email 'github-actions[bot]@users.noreply.github.com'" },
             {
-              name: 'Deploy docs',
-              uses: 'mhausenblas/mkdocs-deploy-gh-pages@master',
```

Comment:
This upstream action gave inconsistent results, second build couldn't push for some reason.